PhpAmqpLib/Wire/AMQPWriter.php

```diff
@@ -180,28 +180,32 @@ public function write_signed_octet($n)
     /**
      * Write an integer as an unsigned 16-bit value
      *
-     * @param $n
+     * @param int $shortValue
      * @return $this
      * @throws \PhpAmqpLib\Exception\AMQPInvalidArgumentException
      */
-    public function write_short($n)
+    public function write_short($shortValue)
```

Comment:
I think the change is good, but we should have the discipline of splitting PRs. Same will happen when you receive contributions later and they will have similar problems. So we have to try at least to set an example (not that I've never did this BTW)